tests/UnpackerTest.php

```diff
@@ -39,7 +39,9 @@ public function testDoNotDuplicateEntry(): void
         @unlink($composerJsonPath);
         file_put_contents($composerJsonPath, '{}');
 
-        $originalEnvComposer = getenv('COMPOSER');
+        $originalEnvComposer = $_SERVER['COMPOSER'];
+        $_SERVER['COMPOSER'] = $composerJsonPath;
+        // composer 2.1 and lower support
```

Comment:
This is unrelated - but due to https://github.com/composer/composer/pull/10218 - the test started failing.

Reply:
can be reverted, I added the change to #832